Create ui-c8y-1022-31-2-the-connect-smartphone-modal-in-the-device-management-quick-links-widget.md (#3730)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-31-2-the-connect-smartphone-modal-in-the-device-management-quick-links-widget.md

---
date: ""

> title: Fixed Connect Smartphone modal not opening from Device Management Quick Links widget

Reviewer: ```suggestion
title: Connect smartphone modal can again be accessed from the Quick links widget
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64411
version: 1022.31.2
---

> The Connect Smartphone modal, which allows users to quickly connect their smartphones to Cumulocity IoT, was not opening when accessed from the Device Management Quick Links widget. This issue has been resolved, and users can now successfully open the Connect Smartphone modal by clicking on the corresponding link in the Quick Links widget. This fix ensures a smoother user experience when connecting smartphones to the platform, enabling users to efficiently manage their devices within the Cumulocity IoT application.

Reviewer: ```suggestion
The Connect smartphone modal, which allows users to quickly connect their smartphones to {{< product-c8y-iot >}}, did not open when accessing it from the "Quick links" widget in the Device Management application. This issue has been resolved, and users can now successfully open the Connect smartphone modal by clicking on the corresponding link in the "Quick links" widget.
```
